internal/cluster/cluster.go

```diff
@@ -101,8 +101,9 @@ type PrepareConfig struct {
 }
 
 type Config struct {
-	PrepareConfig    PrepareConfig
-	MonitorBatchSize int `envconfig:"CLUSTER_MONITOR_BATCH_SIZE" default:"100"`
+	PrepareConfig           PrepareConfig
+	MonitorBatchSize        int  `envconfig:"CLUSTER_MONITOR_BATCH_SIZE" default:"100"`
+	EnableSingleNodeDnsmasq bool `envconfig:"ENABLE_SINGLE_NODE_DNSMASQ" default:"false"`
```

Comment:
I think @romfreiman was thinking of adding an API for it

Reply:
nope, he wasn't. Talked to him

Reply:
Note,
1. Env applies to all users while API is granular and can be set per cluster.
2. When using ENV you can't reflect if the AI is setting the DNS or not, while in API it will be an attribute of the cluster object.
3. By allowing users to enable / disable this option per cluster it can better integrate with the DNS validation that we are planing.
That said, we can start with env and see if the option is even required or it's just going to be always true.

Reply:
@eranco74 i don't think we need api, there is no need for granularity like this. We just want to deploy sometimes without option of adding dnsmasq. If there will be any for api, i will add it.